source/source-connector/configuration-properties/change-stream.txt

```diff
@@ -78,7 +82,32 @@ Settings
 
        |
        | **Default**: ``""``
-       | **Accepted Values**: ``""`` or ``"default"`` or ``"updateLookup"``
+       | **Accepted Values**: ``""``, ``"updateLookup"``, ``"whenAvailable"``, or ``"required"``
+       
+   * - | **change.stream.full.document.before.change**
+     - | **Type:** string
+       |
+       | **Description:**
+       | Configures the document pre-image your change stream returns on update
+         operations. The pre-image is not available in source records
+         published while copying existing data, and the pre-image configuration
+         has no effect on copying.
+         
+       .. note::
+       
+          To learn about how to configure a collection to enable
+          pre-images, see the :manual:`Server manual entry on pre- and
+          post-images </changeStreams/#change-streams-with-document-pre--and-post-images>`.
+       
+       | The default setting suppresses the document pre-image.
+       | The ``whenAvailable`` setting returns the document pre-image,
+         if available, before the document was replaced, updated, or
```

Comment:
Suggestion:

I found the comma splice made this difficult to read. Consider a slight rewording that I think is easier to comprehend (also slipping in a pronoun to make it sound more relaxed):

"The ``whenAvailable`` setting returns the document pre-image if it's available, before it was replaced, updated, or deleted."